tools/fuchsia/fidl/fidl_library.gni

```diff
@@ -22,6 +22,9 @@ template("fidl_library") {
 
   assert(defined(invoker.sources), "A FIDL library requires some sources.")
 
+  assert(fuchsia_target_api_level != -1,
```

Comment:
are there existing usages of this template?  is fuchsia_target_api_level set for all on them?

Reply:
Yes it's used to make FIDL bindings for Fuchsia SDK libraries (see tools/fuchsia/sdk/sdk_targets.gni). I copied the != -1 assertion from tools/fuchsia/fuchsia_archive.gni. `fuchsia_target_api_level` is a global GN arg set here https://github.com/flutter/engine/blob/main/tools/gn#L570. The build should fail if there's a case where it's not set.